### ISSUE TITLE
 [Spotify Player] Minimize window on command execution Option

### ISSUE THREAD

**Issue by RootRooster** [raycast/extensions#7713]
### Extension

https://www.raycast.com/mattisssa/spotify-player

### Description

The "Minimize Window on Command Execution - Option" feature in the Raycast extension for Spotify allows users to automatically minimize the Spotify window when a command is executed. This feature is designed to optimize the user's workflow by reducing distractions and promoting a focused task-oriented experience while working with the Raycast extension and Spotify simultaneously.

TL;DR: Users can launch Spotify with Raycast and then minimize the window using ﻿`COMMAND + M`. The desire is for Raycast to automate this process.

### Who will benefit from this feature?

The users.

**Comment by peduarte** [comment]
Thanks, that's a good idea. The Spotify App is only ever launched once. Once i't running in the background, it shouldn't be brought up to front again. We need to use AppleScript to launch the App, and it's a bit tricky to know when it has successfully launched, so I'm not sure how doable this actually is. 

I don't have the capacity to look into this right now, but will leave this open in case someone wants to tackle it, I'd be happy to review the PR
